test/TestTemplate.hs

```diff
@@ -47,9 +47,9 @@ makeInputTest :: TestTree
 makeInputTest =
   testGroup
     "make input"
-    [ testCase "empty" $ L.makeInput (L.Separator " ") "" @?= L.Input "" [""]
+    [ testCase "empty" $ L.makeInput (L.Separator " ") "" @?= L.Input "" [""] 1
     , testCase "separator ','" $
-        L.makeInput (L.Separator ",") "a, b, c" @?= L.Input "a, b, c" ["a", " b", " c"]
+        L.makeInput (L.Separator ",") "a, b, c" @?= L.Input "a, b, c" ["a", " b", " c"] 3
```

Comment:
Looks like that `makeInput` should be used at the right side of `@?=`. This will make such tests more durable.